build.psm1

```diff
@@ -696,3 +716,8 @@ function Copy-CrossCompatibilityModule
         }
     }
 }
+
+function Export-NuPkg
```

Comment:
This new function is not used and because it is so simple, I would not bother with an abstraction layer

Reply:
I had written it for future-proofing, but I agree, it's not needed